DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -275,7 +275,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="18">
+  <class name="pat::PackedCandidate" ClassVersion="19">
```

Comment:
this comment still holds
last time I somehow thought that this change in PackedCandidate also came with a b-tagging PR for re-miniAOD.
In fact, the reason for the change of class version  in 75X was  https://github.com/cms-sw/cmssw/pull/8559

Anyways, @nhanvtran , this file can't be changed this way:
- the "easy" way is to put the version 19, 20, and 21 version-checksum pairs from 75X/76X here, add version 22-checksum=691064527 here and also forward port it to 75X/76X
- the less easy way is to backport the https://github.com/cms-sw/cmssw/pull/8559 piece as well (hoping that only PackedCandidate can be update, without pulling in everything else)
@Dr15Jones please advise